content/chapter-03/SystemMetricsLab.md

```diff
@@ -1 +1,40 @@
-# Lab: common tools for system metrics
\ No newline at end of file
+# Lab: common tools for system metrics
+
+* In this lab, two Virtual Machine instances will be created: One will run a stress test, and the other will run the application to be tested.
+
+* Tester instance
+  * Create a small instance to launch a stress test tool. e.g.: AWS EC2 t2.micro, AMI: Ubuntu Server 16.04 LTS (HVM), SSD Volume Type - ami-66506c1c
+  * Configure that instace to have 8GB of storage.
+  * ssh to the instance and install [Apache Bench](https://en.wikipedia.org/wiki/ApacheBench):
```

Comment:
ssh into the instance to install [Apache Bench](https://en.wikipedia.org/wiki/ApacheBench):

Reply:
Done